HT4_part2.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+initial_text = r"""homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+
+
+def count_whitespaces_func(txt):
+    whitespace_cnt = re.findall('\s', txt)
+    return len(whitespace_cnt)
+
+def misspelling_func(incorrect_word, correct_word):
+    text_replace_iz = initial_text.lower().replace(' ' + incorrect_word + ' ', ' ' + correct_word + ' ')
+    return text_replace_iz
+
+def last_words_func(txt):
+    last_word_list = re.findall('(\w+)[:.|!?…]', txt)
+    return last_word_list
+
+def capitalize_append_sent_func(txt_list, paragraph_num):

```

Comment:
I think this function is a bit wider than just "capitalize", let's come up with some more specific name?